docs/decisions/0007-access-control-of-events.rst

```diff
@@ -0,0 +1,51 @@
+Access control of events
+========================
+
+Status
+------
+
+Pending
+
+Context
+-------
+
+`event-routing-backends` can transform filtered edX events in xAPI and Caliper format which can then be routed to configured URLs. Procedure for configuring URLs of learning record consumers can be found in the `documentation`_. A method is required to filter edX events based on event name, course id, organisation id etc. or on any combination of these properties.
+
+Decision
+--------
+
+#. A method will be created that filters events that match the criteria specified in `match_params` in router configurations.
+
+#. `match_params` is a dict comprising of one or more key value pairs.
```

Comment:
[clarification] Can clarify whether the matching is happening against the edX event schema or the xAPI/Caliper event schema?  If the latter, the schema may be different for each standard, correct?